app/views/photos/index.html.erb

```diff
@@ -1,10 +1,22 @@
 <% photos.each_with_index do |photo, i| %>
+<% if authorized?(photo) %>
   <h3>
     Photo <%= i + 1 %>
   </h3>
+<<<<<<< HEAD
```

Comment:
Please resolve this merge conflict,  also please remove `<%= flash[:alert] = "You do not have permission to view this page."%>` and just show an empty span please